util/template/expression_template.go

```diff
@@ -0,0 +1,47 @@
+package template
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/Masterminds/sprig"
+	"github.com/antonmedv/expr"
+	"github.com/doublerebel/bellows"
+)
+
+const kindExpression kind = "="
+
+func init() {
+	if os.Getenv("EXPRESSION_TEMPLATES") != "false" {
+		kinds = append(kinds, kindExpression)
+	}
+}
+
+func expressionReplace(w io.Writer, expression string, env map[string]interface{}, allowUnresolved bool) (int, error) {
```

Comment:
Tests for this file?

Reply:
these are covered by `replace_test.go` - let me add coverage output